### PR TITLE
Migrate Jenkinsfile to GH Actions Workflow

### PR DESCRIPTION
**Pull request by reakaleek**
## Summary

Migrate Jenkinsfile to GH Actions Workflow

## Tasks
- [x] Migrate Jenkinsfile
- [x] Delete Jenkins Job config
- [ ] Delete Jenkins Job
- [ ] Update sheet

### CODE REVIEW

**Comment by v1v** [comment]
Do you mind to include the opentelemetry github action similarly done in https://github.com/elastic/ecs-logging-nodejs/pull/134? 🙏

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ repos:
     -   id: check-executables-have-shebangs
     -   id: check-merge-conflict
 
--   repo: git@github.com:elastic/apm-pipeline-library
+-   repo: https://github.com/elastic/apm-pipeline-library
     rev: current
     hooks:
     -   id: check-bash-syntax
```
Comment (v1v) [inline .pre-commit-config.yaml:12]: L13 and L14 are not needed could you update this file 🙏 ?

**Comment by reakaleek** [comment]
> Do you mind to include the opentelemetry github action similarly done in [elastic/ecs-logging-nodejs#134](https://github.com/elastic/ecs-logging-nodejs/pull/134)? 🙏

Sure, this PR is kinda on hold for now. But I will apply the requested changes when I come back to this.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (reakaleek) [reply .pre-commit-config.yaml:12]: https://github.com/elastic/ecs-logging-php/pull/41/commits/7dcd5384604b23918489d56b99d299996c081d9f

**Comment by reakaleek** [comment]
tests are also failing on the main trunk. Will be fixed here: https://github.com/elastic/ecs-logging-php/pull/45

**Review by v1v — APPROVED** [review]

**Review by SergeyKleyman — APPROVED** [review]
It looks good (I ran a few tests in https://github.com/elastic/ecs-logging-php/pull/48).
Thank you very much for all the work.

**Comment by reakaleek** [comment]
> It looks good (I ran a few tests in #48). Thank you very much for all the work.

Thank you for verifying!

**Pull request merged**